end-to-end-tests/cypress/e2e/OSCAL/url_navigation.cy.js

```diff
@@ -0,0 +1,95 @@
+const CATALOG_NAVIGATION =
+  "NIST Special Publication 800-53 Revision 5: Security and Privacy Controls for Federal Information Systems and Organizations";
+const CATALOG_URL_NAV =
+  "http://localhost:8080/catalog/613fca2d-704a-42e7-8e2b-b206fb92b456";
```

Comment:
The functions need to use the `base_url` variable and be relative to that

Reply:
There is a way to configure a baseURL in the config file. <a href ="https://docs.cypress.io/api/commands/visit#Prefixes"> Configuring baseURL</a>
Would that be something worth looking into?
It should make code look cleaner because we can just call `cy.visit('/')` to navigate to the base url

Reply:
> There is a way to configure a baseURL in the config file. 
Nevermind, I just saw it's already defined